src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx

```diff
@@ -38,12 +40,15 @@ export const ActionSelectionForm = ({
   canChangeHolder,
   canEndorseBeneficiary,
   isSurrendered,
+  isLoading,
```

Comment:
rather than putting isLoading everywhere, would it make more sense to put in a placeholder above them?
```
<LoadingPlaceholder isLoading>
  {isTitleEscrow ? <TitleEscrowAssetManagementForm /> : null}
  {isWallet ? "WTF?" : null }
</LoadingPlaceholder>
```